scripts/injectPlugin.js

```diff
@@ -97,27 +73,17 @@ const conditionalReplace = (node, file, options) => {
 };
 
 function transform(options) {
-  return (tree, file) => {
+  return (tree) => {
     visit(tree, ["paragraph", "emphasis", "heading"], (node) => {
-      conditionalReplace(node, file, options);
-    });
-
-    visit(tree, "text", (node) => {
-      unconditionalReplace(node, file, options);
-    });
-
-    visit(tree, "code", (node) => {
-      unconditionalReplace(node, file, options);
+      conditionalReplace(node, options);
     });
   };
 }
 
 function injectPlugin(param) {
-  const dictPath = path.resolve(__dirname, "computed-dict.json");
-  const dict = JSON.parse(fs.readFileSync(dictPath, "utf8"));
   return [
     transform,
-    { dict, isPolkadot: param.isPolkadot, dryRun: false, verbose: false, debug: false },
+    { isPolkadot: param.isPolkadot, dryRun: false, verbose: false, debug: false },
```

Comment:
all the changes in this file basically remove the old functionality that swapped in the dictionary values but I left the functionality for conditional rendering.